.github/workflows/linux_qemu.yml

```
@@ -141,7 +141,8 @@ jobs:
           rm -f /usr/bin/ld.bfd && ln -s /host/usr/bin/${TOOLCHAIN_NAME}-ld.bfd /usr/bin/ld.bfd &&
           rm -f /usr/bin/ninja && ln -s /host/usr/bin/ninja /usr/bin/ninja &&
           git config --global --add safe.directory /numpy &&
-          python -m pip install -r /numpy/requirements/build_requirements.txt &&
+          grep -v ninja /numpy/requirements/build_requirements.txt > /tmp/build_requirements.txt &&
```

Use the host ninja for the build
```suggestion
          # No need to build ninja from source, the host ninja is used for the build
          grep -v ninja /numpy/requirements/build_requirements.txt > /tmp/build_requirements.txt &&
```